Altis_Life.Altis/core/gangs/fn_clientGangLeft.sqf

```diff
@@ -0,0 +1,10 @@
+/*
+    File: fn_clientGangLeft.sqf
+    Author: Bryan "Tonic" Boardwine
+
+    Description: leaves the group
+*/
+
+life_my_gang = objNull;
```

Comment:
objNull v grpNull?

Reply:
pretty sure I didn't touch that, will fix it though :D